tui_list.go

```diff
@@ -29,12 +30,16 @@ func newKeyMap() *keyMap {
 			key.WithHelp("enter", "merge (rebase)"),
 		),
 		mergeDefault: key.NewBinding(
-			key.WithKeys("ctrl+enter"),
-			key.WithHelp("ctrl+enter", "merge (merge-commit)"),
+			key.WithKeys("ctrl+m"),
+			key.WithHelp("ctrl+m", "merge (merge-commit)"),
```

Comment:
afaik this is inaccurate - merge commit is not always the default?

Reply:
The default is using merging using "--rebase". This refers to merging with an additional merge commit iirc

Reply:
allright